src/Veldrid.ImGui/ImGuiRenderer.cs

```diff
@@ -259,33 +259,33 @@ public void ClearCachedImageResources()
             switch (factory.BackendType)
             {
                 case GraphicsBackend.Direct3D11:
-                {
-                    if (stage == ShaderStages.Vertex && colorSpaceHandling == ColorSpaceHandling.Legacy) { name += "-legacy"; }
-                    string resourceName = name + ".hlsl.bytes";
-                    return GetEmbeddedResourceBytes(resourceName);
-                }
+                    {
```

Comment:
Can you please modify the .editorconfig so that this indentation is the standard?